src/sentry/web/frontend/auth_login.py

```diff
@@ -82,11 +209,32 @@ def get_auth_provider(self, organization_slug):
 
         return auth_provider
 
-    def get_login_form(self, request: Request):
-        op = request.POST.get("op")
-        return AuthenticationForm(request, request.POST if op == "login" else None)
+    def handle_register_form_submit(
+        self, request: Request, context: dict, organization: Organization, **kwargs
+    ) -> HttpResponse:
+        """Validates a completed register form, redirecting to the next
```

Comment:
nit:
`"""`'s on their own line

Reply:
Can we also call out that "register" here means accepting invite link?

Reply:
Once you accept the invite link there's a form you have to fill out that confirms actual registration, so accepting the invite link is only one part of registration.